openapi/nexus.json

```diff
@@ -13706,22 +13706,37 @@
         ]
       },
       "Sled": {
-        "description": "Client view of a [`Sled`]",
+        "description": "An operator's view of a sled.",
         "type": "object",
         "properties": {
           "baseboard": {
             "$ref": "#/components/schemas/Baseboard"
           },
+          "cpus": {
+            "description": "The number of online processors available to the sled.",
+            "type": "integer",
+            "format": "int64"
+          },
           "id": {
             "description": "unique, immutable, system-controlled identifier for each resource",
             "type": "string",
             "format": "uuid"
           },
+          "physical_ram": {
+            "description": "The total amount of physical RAM available to the sled.",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/ByteCount"
+              }
+            ]
+          },
           "rack_id": {
+            "description": "The rack to which this Sled is currently attached.",
             "type": "string",
             "format": "uuid"
           },
           "service_address": {
+            "description": "The underlay address on which the Sled Agent is listening.",
```

Comment:
I realize this is just being updated to be more descriptive, but why is this in the external API? We can defer to a separate review of the JSON if useful.